hudi-client/hudi-spark-client/src/main/java/org/apache/hudi/client/SparkRDDWriteClient.java

```diff
@@ -442,25 +467,10 @@ private void completeTableService(TableServiceType tableServiceType, HoodieCommi
     } else {
       writeTimer = metrics.getDeltaCommitCtx();
     }
+    table.getHoodieView().sync();
```

Comment:
need to discuss on whats the right thing to do here. timeline does not refresh for every operation, but in some tests we do N no of operations and so we need this refresh. We might have a deltastreamer which uses same writeClirnt and does N no of write operations. For spark datasource, should not be an issue.